rest/config.go

```diff
@@ -407,6 +407,7 @@ const (
 	SyncFunction     JSLoadType = iota // Sync Function JavaScript load.
 	ImportFilter                       // Import filter JavaScript load.
 	ConflictResolver                   // Conflict Resolver JavaScript load.
+	WebhookFilter                      // Webhook filter JavaScript load.
```

Comment:
I think we're missing a string representation of the new `WebhookFilter` const below in `String()`. Would be worth adding coverage of this in a test to prevent the same happening in the future, as it's an easy thing to miss.

Check out `TestLogLevelNames` for an equivalent.

Reply:
A new test has been included: TestJSLoadTypeString